### PR TITLE
storage: Remove support for Stratis 2

### PR DESCRIPTION
**Pull request by mvollmer**

### CODE REVIEW

**Comment by mvollmer** [comment]
Yay, looks like we can actually remove stratis 2 support.

File: pkg/storaged/stratis/create-dialog.jsx [hunk 1]
```diff
@@ -128,7 +124,10 @@ export function create_stratis_pool() {
                         let clevis_info = null;
                         if (adv)
                             clevis_info = ["tang", JSON.stringify({ url: vals.tang_url, adv })];
-                        return client.stratis_create_pool(vals.name, devs, key_desc, clevis_info)
+                        return client.stratis_manager.CreatePool(vals.name,
+                                                                 devs,
+                                                                 key_desc ? [true, key_desc] : [false, ""],
+                                                                 clevis_info ? [true, clevis_info] : [false, ["", ""]])
```
Comment (cockpituous) [inline pkg/storaged/stratis/create-dialog.jsx:130]: These 2 added lines are not executed by any test.

File: pkg/storaged/stratis/pool.jsx [hunk 2]
```diff
@@ -116,7 +116,7 @@ function create_fs(pool) {
         Action: {
             Variants: action_variants,
             action: function (vals) {
-                return client.stratis_create_filesystem(pool, vals.name, vals.size)
+                return pool.CreateFilesystems([[vals.name, vals.size ? [true, vals.size.toString()] : [false, ""]]])
```
Comment (cockpituous) [inline pkg/storaged/stratis/pool.jsx:119]: This added line is not executed by any test.

**Review by martinpitt — APPROVED** [review]
Thanks! :broom: 

We need to be aware that with each of these cleanups we cut off another piece of functionality when connecting to a RHEL 8 machine with Cockpit Client in beiboot mode. However, we already stopped running RHEL 8 tests a while ago, so (1) untested code == broken, and (2) I also know that @allisonkarlitskaya wants to get rid of some Python 3.6-isms. I'm ok with only supporting connecting to RHEL 8 when cockpit-system/bridge are installed on these machines.

**Pull request merged**